src/backend/joanie/core/factories.py

```diff
@@ -161,6 +161,8 @@ class Meta:
     username = factory.Faker("user_name")
     email = factory.Faker("email")
     password = make_password("password")
+    first_name = factory.Faker("first_name")
+    last_name = factory.Faker("last_name")
```

Comment:
I have to confirm this point but from OpenEdX we could not have `first_name` and `last_name` but a `full_name` instead.

Reply:
I confirm from OpenEdX I can retrieve `user.profile.name` which is the full name. `user.first_name` and `user.last_name` exists but are not used in OpenEdX so those fields are always blank.

Reply:
ok I will manage it in an other PR, to clean all everywhere.

Reply:
Here is the link to the PR I add this property to the JWT claim
https://github.com/openfun/fonzie/pull/26